repose-aggregator/commons/utilities/src/main/java/org/openrepose/commons/utils/servlet/http/HeaderInteractor.java

```diff
@@ -50,25 +50,26 @@
     List<String> getSplittableHeader(String headerName);
 
     /**
-     * Returns the header value with the highest quality. The default quality is 1.0 if unspecified.
+     * Returns the header value(s) with the highest quality. The default quality is 1.0 if unspecified.
      * This treats each header entry as if it's one value, should there be multiple values per header entry
      * exceptions are likely to occur.
      *
      * @param headerName the name of the header to get the preferred value for
-     * @return the highest qualitied value
+     * @return the highest quality value(s)
      * @throws QualityFormatException when quantity exists but cannot be parsed
      */
-    String getPreferredHeader(String headerName);
+    List<String> getPreferredHeader(String headerName);
```

Comment:
we should pluralize the method name since it's now returning a list? for all three methods